components/constant_contact/actions/add-contact/add-contact.mjs

```diff
@@ -13,6 +14,7 @@ export default {
         constantContact,
         "emailAddress",
       ],
+      reloadProps: true,
```

Comment:
Is `reloadProps` needed here? I don't see `emailAddress` being used in `additionalProps()`.

Reply:
Hi @michelle0927, It's just to load the customFields. I know they are not related to emailAddress but emailAddres is the only required so I thought it would be good to use it to fire the customFields